persistence-modules/spring-persistence-simple-2/pom.xml

```diff
@@ -32,6 +32,13 @@
             <version>${h2.version}</version>
             <scope>test</scope>
         </dependency>
+        
```

Comment:
I looks like we have got some tabs in the pom.xml here.

Reply:
Done!